src/pages/elements/pictograms/contribute.mdx

```diff
@@ -23,9 +23,9 @@ and work well in presentations and marketing communications.
 
 Don’t see the pictogram you need in the library? Make your own! Follow these guidelines to ensure visual consistency and proper formatting.
 
-- All pictograms should be unique and not redundant with any existing pictograms in the system. Look through the pictograms in the IBM Design Sketch kit to ensure that your proposed new pictogram is not already represented.
-- All pictograms should adhere to the IBM Design Language visual style.
-- All pictograms should comply with IBM accessibility standards.
+- All pictograms should be unique and not redundant with any existing pictograms in the system. Look through the pictograms in the [IBM Design Sketch kit](https://ibm.com/standards/web/design-kit) to ensure that your proposed new pictogram is not already represented.
+- All pictograms should adhere to the [IBM Design Language visual style](https://w3.ibm.com/design/language/elements/pictograms/).
```

Comment:
```suggestion
- All pictograms should adhere to the [IBM Design Language visual style](/elements/pictograms/design).
```

The link on the live site is wrong, they had some hardcoded links to the old internal url 🤦‍♀